### DISCUSSION TITLE
Looking for an element in a column of arrays

### DISCUSSION THREAD

**Discussion by theojolliffe** [pocketbase/pocketbase#1803] [Q&A] [answered]
Is there any way using getList (or otherwise) to check which records hold an array containing a specified element.
I've been using ~, for instance `filter: 'ingredients_match ~ "chicken"',`
But this returns arrays with chicken_stock, while I only want "chicken".
Thanks

**Comment by ganigeorgiev** [comment, marked as answer]
Hm, if `ingredients_match` is a multi-`select` field, then you can try:

- use the `~` operator but wrap your string with double quotes (_multi-valued fields are always stored as serialized json_):
  ```js
  filter: `ingredients_match ~ '"chicken"'`
  ```
  
- combine the `?=` (at least one of equal) operator with the the `:each` modifier to apply condition over each value individually:
  ```js
  filter: `ingredients_match:each ?= "chicken"`
  ```

**Reply by theojolliffe** [reply]
Amazing thank you. I had ingredients_match as a json field but wrapping in double quotes seems to have worked.
